rpmlint/descriptions/PolkitCheck.toml

```diff
@@ -0,0 +1,16 @@
+polkit-user-privilege="""
+The package allows unprivileged users to carry out privileged
+operations. This could cause security
+problems if not done carefully. #REVIEW_NEEDED_TEXT#
+"""
+polkit-untracked-privilege="""
+The privilege is not listed in /etc/polkit-default-privs.*
+which makes it harder for admins to find. Furthermore improper polkit
+authorization checks can easily introduce security issues. #REVIEW_NEEDED_TEXT#
+"""
+polkit-cant-acquire-privilege="""
```

Comment:
Instead of providing an empty text here I think it is better to completely remove the `add_info` invocation above. You can instead keep the `if` branch with a comment like:

```
if found_no:
    # in the past this triggered 'polkit-cant-acquire-privilege', but this is not considered very helpful for
    # packagers since there are legitimate use cases for this.
    pass
```

We can then drop this `found_no` logic in a follow-up commit completely, just so that it is documented in the check's git history.